Dockerfile

```diff
@@ -33,40 +33,29 @@ RUN apk update \
                         --infodir=/usr/share/info \
          && make && make install \
          && rm -rf /tmp/zeromq* && rm -rf /tmp/czmq* \
-         && rm -rf /var/cache/apk/* \
-    && echo " ... installing api-gateway-zmq-adaptor" \
-         && curl -L https://github.com/adobe-apiplatform/api-gateway-zmq-adaptor/archive/${ZMQ_ADAPTOR_VERSION}.tar.gz -o /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION} \
-         && apk update \
-         && apk add check-dev \
-         && cd /tmp/ \
-         && tar -xf /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION} \
-         && cd /tmp/api-gateway-zmq-adaptor-* \
-         && make test \
-         && mkdir -p /usr/local/sbin \
-         && PREFIX=/usr/local/sbin make install \
-         && rm -rf /tmp/api-gateway-zmq-adaptor-* \
-         && apk del check-dev \
-    && apk del gcc tar libtool make musl-dev openssl-dev g++ zlib-dev curl \
-    && rm -rf /var/cache/apk/*
+         && rm -rf /var/cache/apk/* 
 
 RUN apk update \
     && apk add libgcc libstdc++
 
-# --- DEV ONLY ---
-# RUN apk update \
-#    && apk add \
-#           gcc tar libtool zlib make musl-dev openssl-dev g++ zlib-dev curl \
-#    && apk add check-dev
-# COPY src /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION}/src
-# COPY tests /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION}/tests
-# COPY Makefile /tmp/api-gateway-zmq-adaptor-${ZMQ_ADAPTOR_VERSION}/Makefile
-# RUN cd /tmp/api-gateway-zmq-adaptor-* \
-#         && make test \
-#         && mkdir -p /usr/local/sbin \
-#         && PREFIX=/usr/local/sbin make install \
-#         && rm -rf /tmp/api-gateway-zmq-adaptor-*
+# --- DEV ---
+RUN apk update \
```

Comment:
I guess we could leave this uncommented on the premise that this sidecar process runs within the container anyway and this docker image is used only for development ? Even though another design is to deploy a side-container that runs alongside the api gateway container, communicating via [ipc](https://docs.docker.com/engine/reference/run/#ipc-settings---ipc) .

Reply:
Q: "this docker image is used only for development ?" 
A: not yet, but trying to make it modular